src/platform-implementation-js/namespaces/app-menu.ts

```diff
@@ -3,11 +3,12 @@ import { AppMenuItemView } from '../views/app-menu-item-view';
 
 export type AppMenuItemDescriptor = {
   name: string;
-  iconElement: string;
+  iconUrl: string;
```

Comment:
just a note that we probably should change documentation too after these changes

Reply:
is this is a breaking change, once the inboxsdk version is deployed will this change influece existing code in production (ours and other people that use inboxsdk?)